Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'pg', '~> 1.1'
 gem 'puma', '~> 5.0'
 gem 'rails', '~> 6.1.3', '>= 6.1.3.1'
 gem 'sass-rails', '>= 6'
+gem 'sendgrid-ruby'
```

Comment:
Use the pessimistic operator(~>) to set a version range for this gem. This will save you from unwanted gem updates if something happens to the gemfile.lock.